3-methyl-heptane/Wang_phi_0.5_m-h_0.0122_xO2_0.2075_NUIG.yaml

```diff
@@ -0,0 +1,148 @@
+---
+file-authors:
+    - name: Nathan Harms
+      ORCID: 0000-0003-2680-371X
+file-version: 0
+chemked-version: 0.0.1
```

Comment:
`chemked-version` should be 0.4.1 if these files are compatible with that version (if they're not, they should be made to be).